AnnoyancesFilter/Other/sections/annoyances.txt

```diff
@@ -8,6 +8,7 @@
 !######### Annoying elements (not ads)
 ! SECTION: Other annoyances - Uncategorized
 !
+jam-news.net##section#donationPopup.donation-popup-box.active
```

Comment:
I think that it can be simplified to:
```adblock
jam-news.net###donationPopup
```

Please also move it to `Donation widgets` section, somewhere here:
https://github.com/AdguardTeam/AdguardFilters/blob/2028b36df3aac7d2e74591ecfe53fbd2421ce60a/AnnoyancesFilter/Widgets/sections/widgets.txt#L2107-L2110